File: cmstestsuite/tasks/batch_fileio_managed/code/grader.java
import java.util.Scanner;
import java.io.PrintWriter;
import java.io.File;
import java.io.FileNotFoundException;

public class grader {

    public static void main(String[] args) throws FileNotFoundException {
        File file = new File("input.txt");
        Scanner scanner = new Scanner(file);
        int n = scanner.nextInt();
        PrintWriter writer = new PrintWriter("output.txt");
        writer.println("correct " + batchfileiomanaged.userfunc(n));
        writer.close();
    }
}

File: cmstestsuite/tasks/batch_fileio_managed/code/grader.cs
using System;
using System.IO;

public static class Grader
{
    public static void Main()
    {
        StreamReader reader = new StreamReader("input.txt");
        int n = int.Parse(reader.ReadLine());
        StreamWriter writer = new StreamWriter("output.txt");
        writer.WriteLine("correct " + Batchfileiomanaged.Userfunc(n));
        reader.Close();
        writer.Close();
    }
}
